lisa/sut_orchestrator/azure/features.py

```diff
@@ -1206,6 +1206,12 @@ def get_all_nics_ip_info(self) -> List[IpInfo]:
 @dataclass()
 class AzureDiskOptionSettings(schema.DiskOptionSettings):
     has_resource_disk: Optional[bool] = None
+    cached_disk_size: search_space.CountSpace = field(
```

Comment:
Can it be `os_disk_size`, and put it in common DiskOptionSettings?

Reply:
os_disk_size might be not accurate in describing the cached disk size which is related to VM size.

Reply:
`os_disk_size` is a requirement on the disk settings. The cached disk size is calculated in Azure, based on os_disk_size.

Reply:
Done